metadata.yaml

```diff
@@ -46,22 +46,3 @@ resources:
     type: oci-image
     description: Container image for Prometheus
     upstream-source: docker.io/ubuntu/prometheus@sha256:03936a740851b8786c328e08803a70feaa97731714a8dbe4b709c44366699b0c
-  promql-transform-amd64:
-    # When picking which version to use with the charm, use the latest one from
-    # https://github.com/canonical/promql-transform that has the same version number
-    # as the Prometheus shipped within the `prometheus-image` resource.
```

Comment:
Adapt these comments for `charmcraft.yaml`?

Reply:
This bears a conversation with @simskij about how important this alignment is. If it's crucial, then at the very least this doc should be improved enough to indicate _how_ to check what's in the image. If it's vital, then it should be automatically populated

Reply:
it's really not **that** important for the transformation tool, as all we do with it is to inject label selectors, which most likely won't change much (if ever)